rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -883,13 +918,13 @@ audio_port* cell_audio_thread::open_port()
 	return nullptr;
 }
 
-template <AudioChannelCnt downmix>
+template <AudioChannelCnt channels, AudioChannelCnt downmix>
```

Comment:
Unjustified template argument addition, just use normal argument. It only increases compilation time for no real performance benefit.

Reply:
I don't think so

Reply:
but it would certainly reduce the switch case hydra